pallets/order-book/src/storage_data_layer.rs

```diff
@@ -170,6 +171,17 @@ impl<T: Config> DataLayer<T> for StorageDataLayer<T> {
         }
     }
 
+    fn get_all_limit_orders(
```

Comment:
I'm not sure if it's good idea to load all orders for order book into memory. We should benchmark it with almost full order book and make some decision according to results. Same for cache storage

Reply:
Ok, will do that in benchmarking stage